encode/.htaccess

```diff
@@ -0,0 +1,10 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^ontology$ https://encode.uni-hamburg.de/ontology [R=302,L]
+RewriteRule ^vocabulary$ https://encode.uni-hamburg.de/vocabulary [R=302,L]
+RewriteRule ^modules/?$ https://encode.uni-hamburg.de/modules/ [R=302,L]
+RewriteRule ^vocabulary/?$ https://encode.uni-hamburg.de/competences/$1 [R=302,L]
+RewriteRule ^openings/?$ https://encode.uni-hamburg.de/openings/$1 [R=302,L]
+RewriteRule ^modules/?$ https://encode.uni-hamburg.de/modules/$1 [R=302,L]
+RewriteRule ^competences/(.+)$ https://encode.uni-hamburg.de/competences/$1 [R=302,L]
+RewriteRule ^openings/(.+)$ https://encode.uni-hamburg.de/openings/$1 [R=302,L]
```

Comment:
Some of the above look wrong, please check.  There are duplicates and mismatched names and $1 with no patterns.